core/src/err.rs

```diff
@@ -75,10 +75,8 @@ pub enum Error {
         "The ODBC driver version `{0}` is incompatible with libmongosqltranslate version `{1}`"
     )]
     LibmongosqltranslateLibraryIsIncompatible(&'static str, String),
-    #[error(
-        "The libmongosqltranslate command `{0}` failed. Error message: `{1}`. Error is internal: {2}"
-    )]
-    LibmongosqltranslateCommandFailed(&'static str, String, bool),
+    #[error("The libmongosqltranslate command `{0}` failed. Error message: `{1}`.")]
+    LibmongosqltranslateCommandFailed(&'static str, String),
```

Comment:
```suggestion
    #[error("The mongosql command `{0}` failed. Error message: `{1}`.")]
    MongosqlCommandFailed(&'static str, String),
```
I think we should rename all the `libmongosqltranslate` stuff to `mongosql`. I noticed there's other code that can also be renamed to `mongosql`. I think it would be good to convert all instances of `mongosqltranslate` and `libmongosqltranslate` to `mongosql`. Also, I believe the error above this one and several below it can be removed since they were only used by mongosqltranslate.